### PR TITLE
print package details at end

### PR DESCRIPTION
**Pull request by esc**
One of the most common questions when running integration tests is:
"what were the final package versions that you ran with?". Since this
is so common, we now print the final versions of the relevant packages
at the end of the test-run.

### CODE REVIEW

**Review by stuartarchibald — DISMISSED** [review]
Thanks for the patch.

**Comment by esc** [comment]
@stuartarchibald the implementation now searches the captured output of the command with Python.

File: texasbbq.py [hunk 1]
```diff
@@ -437,6 +437,16 @@ def print_environment_details(target):
     execute("conda list -n {}".format(target.name))
 
 
+def print_package_details(source_name, target_name):
+    lines= execute("conda list -n {}".format(target_name),
+                   capture=True).decode("utf-8").split("\n")
```
Comment (stuartarchibald) [inline texasbbq.py]:
```suggestion
    lines = execute("conda list -n {}".format(target_name),
                    capture=True).decode("utf-8").split("\n")
```

**Review by stuartarchibald — COMMENTED** [review]
Thanks for fixing this to not use the shell, style typo else looks good.

File: texasbbq.py (hunk 1 above)
Comment (esc) [reply texasbbq.py]: Yeah, texasbbq needs to be developed outside of a conda-env to ensure an existing, activated env doesn't clobber the envs activated by the tool itself. I didn't check to see if flake8 was installed so I let this one slip.

**Pull request merged**